webprojects/fundraising.md

```diff
@@ -0,0 +1,27 @@
+# Fundraising website: how to localize and test
+
+[donate.mozilla.org](https://donate.mozilla.org) is our platform to collect donations. The main form is for Mozilla Foundation, but it also hosts a separate form for donations to Thunderbird.
+
+This website relies on third parties to handle payment. The localization of the payment form for debit/credit card payment is managed by Stripe and we do not have control.
```

Comment:
Maybe simpler: "The localization of the payment system for debit/credit cards is managed by…"